### DISCUSSION TITLE
How to use slicing with one shot learning

### DISCUSSION THREAD

**Discussion by HarshVardhanGoyal** [obss/sahi#667] [Q&A] [answered]
Hey, thanks for the very well explained implementation, I wanted to implement this but for one-shot object detection. Can you point out the changes this repo will require?

**Comment by fcakyon** [comment, marked as answer]
Hello @HarshVardhanGoyal, `sahi` can support any detector following these steps: https://github.com/obss/sahi#contributing

Check this repo on how to implement zero shot object detector with `sahi`: https://github.com/machinelearnear/large-scale-object-detection-with-sahi-detectron2

**Comment by HarshVardhanGoyal** [comment]
Thanks for the quick response, this is really helpful!
